templates/main.php

```diff
@@ -79,6 +82,7 @@
 					<a id="tracking-settings">Location tracking settings</a>
 			</div>
 		</div>
+		<?php endif; ?>
```

Comment:
This ending tag needs to be further up, otherwise it also hides the settings area.